mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -17,13 +17,12 @@ impl BridgeController {
         Self::default()
     }
 
-    pub async fn start(&mut self) -> Result<()> {
+    pub async fn start(&mut self, system_address: String, device_id: &str) -> Result<()> {
```

Comment:
At some point we will need to return a shutdown handle and integrate properly into the startup logic

Reply:
agree, shutdown is not handled yet by bridge